### PR TITLE
Update IX bidder to support all MediaTypes.

### PR DESCRIPTION
**Pull request by DGarbar**
Refactored code.
Url change is not needed.
Left code that fixes empty bid.w and bid.h

### CODE REVIEW

**Comment by DGarbar** [comment]
Also they added passing of the request type for Legacy IxAddapter. 
This change require refactoring of our code logic. (To pass additional information to makeBids method), so it will be ignored due no actual requests to this endpoint

File: src/main/java/org/prebid/server/bidder/ix/IxBidder.java [hunk 1]
```diff
@@ -97,84 +102,85 @@ public IxBidder(String endpointUrl, JacksonMapper mapper) {
...
         }
+
+        return modifiedBidRequests;
     }
 
-    private static List<Format> makeFormatFromBannerWidthAndHeight(Imp imp) {
-        final Banner banner = imp.getBanner();
-        return Collections.singletonList(
-                Format.builder().w(banner.getW()).h(banner.getH()).build());
+    private static Site modifiedSite(Site site, ExtImpIx extImpIx) {
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/bidder/ix/IxBidder.java:131]: i'd suggest to name functions in imperative mood, like `modifySite(..)`
Pls check other fun-s below if agree.

File: src/main/java/org/prebid/server/bidder/ix/IxBidder.java [hunk 2]
```diff
@@ -97,84 +102,85 @@ public IxBidder(String endpointUrl, JacksonMapper mapper) {
...
+                .map(banner -> imp.toBuilder().banner(banner).build())
+                .collect(Collectors.toList());
+    }
 
-        final List<BidRequest> requests = new ArrayList<>();
-        for (Format format : limitedFormats) {
-            final Banner updatedBanner = imp.getBanner().toBuilder()
+    private static List<Banner> modifiedBanners(Banner banner) {
+        final ArrayList<Banner> modifiedBanners = new ArrayList<>();
+        final List<Format> formats = getFormats(banner);
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/bidder/ix/IxBidder.java:158]: Can `getFormats(..)` return null? This may cause NPE if so.

**Review by rpanchyk — CHANGES_REQUESTED** [review]

File: src/main/java/org/prebid/server/bidder/ix/IxBidder.java (hunk 2 above)
Comment (rpanchyk) [reply src/main/java/org/prebid/server/bidder/ix/IxBidder.java:158]: Agree.

**Review by rpanchyk — APPROVED** [review]

**Pull request merged**